target_postgres/tests/test_standard_target.py

```diff
@@ -428,8 +436,10 @@ def test_anyof(postgres_target):
     schema = postgres_target.config["default_target_schema"]
     singer_file_to_target(file_name, postgres_target)
     with engine.connect() as connection:
-        meta = sqlalchemy.MetaData(bind=connection)
-        table = sqlalchemy.Table("commits", meta, schema=schema, autoload=True)
+        meta = sqlalchemy.MetaData()
```

Comment:
Metadata is no longer bound to a connection on init. Now bound on usage on the next line below.